### ISSUE TITLE
Missing functions for ORH (harmonic drive)

### ISSUE THREAD

**Issue by ax-anoop** [SupremeLyy/Motorevo-Studio#1]
Good engineering btw, but please connect with me if you can. I have prototype ORH joints I am testing.

**Comment by ax-anoop** [comment]
https://user-images.githubusercontent.com/95486801/173919057-a159c9f3-8863-49d6-bec8-a9a6bb16a8ca.mp4

Good work supreme, I am a fan.

**Comment by ax-anoop** [comment]
Please contact me if you can.

**Comment by SupremeLyy** [comment]
Hello Gadhrri,

I’m sorry that I reply this mail too late.
Thanks for using our actuator production, if you have any problems with it, please let me know !


Tom
Motorevo

________________________________
发件人: Anoop Gadhrri ***@***.***>
发送时间: Thursday, June 16, 2022 9:16:30 PM
收件人: SupremeLyy/Motorevo-Studio ***@***.***>
抄送: Subscribed ***@***.***>
主题: Re: [SupremeLyy/Motorevo-Studio] Missing functions for ORH (harmonic drive) (Issue #1)


Please contact me if you can.

―

**Comment by ax-anoop** [comment]
No worries Tom, do you have a we-chat I could message you on ? Would be much appreciated.

**Comment by SupremeLyy** [comment]
Yeah, you can contact me with we-chat: 13777826505

发件人: Anoop ***@***.***>
发送时间: 2022年6月23日 12:29
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [SupremeLyy/Motorevo-Studio] Missing functions for ORH (harmonic drive) (Issue #1)


No worries Tom, do you have a we-chat I could message you on ? Would be much appreciated.

―

**Comment by ax-anoop** [comment]
Sent you request supreme!

**Comment by SupremeLyy** [comment]
Sorry, I didn’t receive your message on we-chat.
You can try to add me by scan this picture in we-chat

***@***.***

发件人: Anoop ***@***.***>
发送时间: 2022年6月28日 1:08
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [SupremeLyy/Motorevo-Studio] Missing functions for ORH (harmonic drive) (Issue #1)


Sent you request supreme!

―

**Comment by ax-anoop** [comment]
@SupremeLyy Did not get any picture, want to email it to me ? anoop@axibo.com

**Comment by ax-anoop** [comment]
![image](https://user-images.githubusercontent.com/95486801/176496265-013d18b8-6caf-4565-b721-1b6af77ceb66.png)

**Comment by ax-anoop** [comment]
![WhatsApp Image 2022-06-29 at 1 15 14 PM](https://user-images.githubusercontent.com/95486801/176496960-32c6f079-4ee7-49f6-88fd-d1ae8377bf77.jpeg)
Is this you? i sent friend request.

**Comment by SupremeLyy** [comment]
***@***.***

Can you see the picture now? scan this QR code on your we-chat so you can send a friend application!
发件人: Anoop ***@***.***>
发送时间: 2022年6月30日 1:12
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [SupremeLyy/Motorevo-Studio] Missing functions for ORH (harmonic drive) (Issue #1)


[image]<https://user-images.githubusercontent.com/95486801/176496265-013d18b8-6caf-4565-b721-1b6af77ceb66.png>

―

**Comment by ax-anoop** [comment]
No, i think you are sending wrong picture? This is just a picture of what I sent you.

**Comment by SupremeLyy** [comment]
Yeah, that it me, you can click ADD

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Anoop ***@***.***>
发送时间: 2022年6月30日 1:16
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [SupremeLyy/Motorevo-Studio] Missing functions for ORH (harmonic drive) (Issue #1)


[WhatsApp Image 2022-06-29 at 1 15 14 PM]<https://user-images.githubusercontent.com/95486801/176496960-32c6f079-4ee7-49f6-88fd-d1ae8377bf77.jpeg>
Is this you? i sent friend request.

―

**Comment by ax-anoop** [comment]
![image](https://user-images.githubusercontent.com/95486801/176497944-116c57ed-18d3-44fd-8139-f93a237857c1.png)

I sent multiple times, maybe its not going through ?

**Comment by SupremeLyy** [comment]
Yeah that's me, but I actually receive nothing on my wechat
Let me figure out what is going on my wechat.

发件人: Anoop ***@***.***>
发送时间: 2022年6月30日 1:22
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [SupremeLyy/Motorevo-Studio] Missing functions for ORH (harmonic drive) (Issue #1)


[image]<https://user-images.githubusercontent.com/95486801/176497944-116c57ed-18d3-44fd-8139-f93a237857c1.png>

I sent multiple times, maybe its not going through ?

―

**Comment by ax-anoop** [comment]
Weird.
My cell > +1-647-667-5999 are you able to add me ?
![image](https://user-images.githubusercontent.com/95486801/176499040-597037c6-2b34-4407-ba1d-5b6e6d8b39be.png)

**Comment by SupremeLyy** [comment]
I can search you by QR code and send my application many times, it seems that some thing in the we-chat are blocking us.
Maybe we are doing this too frequent, I will keep trying to contact you tomorrow,
you can send your issue to us on this mail as well, so we may can diagnose what is going wrong on your ORH module.

发件人: Anoop ***@***.***>
发送时间: 2022年6月30日 1:28
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [SupremeLyy/Motorevo-Studio] Missing functions for ORH (harmonic drive) (Issue #1)


Weird.
My cell > +1-647-667-5999 are you able to add me ?
[image]<https://user-images.githubusercontent.com/95486801/176499040-597037c6-2b34-4407-ba1d-5b6e6d8b39be.png>

―

**Comment by ax-anoop** [comment]
I think it worked now, I accepted your request & send message.
